src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentDay = 40;
+  let fullAmount = 0;
+
+  for (let i = 1; i <= days; i++) {
+    fullAmount += rentDay;
+  }
```

Comment:
The loop here is unnecessary because you can calculate the full amount by multiplying the number of days by the daily rate. This will improve the performance of your function.